robot/logging/driver.cpp

```diff
@@ -14,8 +15,68 @@ CAF_BEGIN_TYPE_ID_BLOCK(driver, caf::first_custom_type_id)
 
 CAF_END_TYPE_ID_BLOCK(driver)
 
+namespace app {
+
 constexpr std::string_view component = "app";
 
+/// Logs a message for the `caf.core` component with `debug` severity.
```

Comment:
Copy paste remnants, also below.
```suggestion
constexpr std::string_view component = "app";

/// Logs a message for the `app` component with `debug` severity.
```

Reply:
Yeah, forgot to remove the doxygen docu. Makes no sense to have them in a driver in the first place.